web/client/components/data/query/FilterField.jsx

```diff
@@ -96,7 +96,7 @@ class FilterField extends React.Component {
                         comboFilter={"contains"}/>
                 </div>
                 <div className="filter-field-operator">{selectedAttribute ? this.renderOperatorField() : null}</div>
-                <div className="filter-field-value">{selectedAttribute && this.props.filterField.operator ? this.renderValueField(selectedAttribute) : null}</div>
+                <div className="filter-field-value">{this.props.filterField?.operator === 'isNull' ? null : selectedAttribute && this.props.filterField.operator ? this.renderValueField(selectedAttribute) : null}</div>
```

Comment:
Could we remove this one and place this logic in the [DateField](https://github.com/geosolutions-it/MapStore2/blob/504d686ca9363ff0314dc763b60265a9c94ffa4d/web/client/components/data/query/DateField.jsx#L41) component and use the operator to disable or hide the value field if needed?